Src/Repro/EasyRepro/README.md

```diff
@@ -0,0 +1,122 @@
+﻿# Easy Repro: Skeleton project for reproducing bugs
+
+This project serves as a way to test integration-level changes to the Google
+.NET client libraries, without needing to first publish binaries to NuGet.
+
+This works through a `nuget.config` file that picks up the needed dependencies
+from the local machine. (Instead of using the public NuGet packages.)
+
+This project is not part of the regular build and not ran as part of the
+release testing. It's only for debugging purposes. Read: this is not a
+regression test suite. Rather, it is a debugging tool.
+
+`TODO(chrsmith): Document how to build the Support libraries with DEBUG.`
+
+Read this document top to bottom and be sure to follow along with each
+and every step. Unfortunately there are a lot of subtle things you need
+to get right.
+
+## Getting Started
+
+The first thing to do is to ensure you have a clean enlistment. Create a new
+git branch as applicable and delete any untracked files. (e.g. previously
+built NuGet packages.)
+
+````
+git clean -dxf
+````
+
+## How to Build
+
+To build the .NET Client Libraries, run the following commands from PowerShell.
+You will do this a few times while setting up your local issue repro.
+
+````
+$msbuild14 = "C:\Program Files (x86)\MSBuild\14.0\Bin\MSBuild.exe"
+& $msbuild14 /m /nr:false SupportLibraries.proj
+& $msbuild14 /m /nr:false GeneratedLibraries.proj
+````
+
+## Getting Started
+
+The first thing you need to do is make sure NuGet and the `nuget.config` are
+correct. Edit `nuget.config` and add the file path where you build the client
+libraies. This is dependent on where your repository is located on disk.
+
+`TODO(chrsmith): Confirm this will work for multiple developers.`
+
+````
+notepad .\Src\Repro\nuget.config
+````
+
+Next you should probably delete the local NuGet package caches, since they
```

Comment:
would it be reasonable to have them increment the version instead? then install-package -upgrade will pick up the latest one.